parsers/barcode_128.go

```diff
@@ -22,10 +24,26 @@ func NewBarcode128Parser() *CommandParser {
 
 			if len(parts) > 1 {
 				if v, err := strconv.Atoi(parts[1]); err == nil {
+					// TODO: Add BY command and handle defaults
 					barcode.Height = v
 				}
 			}
 
+			if len(parts) > 2 && len(parts[2]) > 0 {
+				// The default here is 'Y', so we treat anything else than 'N' as true
+				barcode.Line = parts[2][0] != 'N'
```

Comment:
it is probably ok to just use toBoolField here, right?
Even when Y is the default value
It just means that barcode.Line should be set to true when initialized

Reply:
It affects how the invalid value will work, but maybe we should not worry too much about the invalid domain.